tsup.config.ts

```diff
@@ -0,0 +1,13 @@
+import { defineConfig } from "tsup-preset-solid";
+
+export default defineConfig(
+  {
+    entry: "src/index.tsx",
+    devEntry: true,
+  },
+  {
+    // Uncomment this to update package.json
```

Comment:
Curious: What is the use case for this? Why leave it there commented out?

Reply:
so I left it commented out mostly for your information that this is a thing 🤷‍♂️